src/hapi/pipelines/configs/food_security.yaml

```diff
@@ -1,75 +1,36 @@
-# IPC name and description taken from page 53 of
-# https://www.ipcinfo.org/fileadmin/user_upload/ipcinfo/manual/IPC_Technical_Manual_3_Final.pdf
+food_security:
+  adm_ignore_patterns:
+    - " urban"
+    - " rural"
+    - "rural "
+    - "(1)"
+    - "(2)"
+    - "(3)"
+    - "(4)"
+    - "(5)"
+    - "_1"
+    - "_2"
+    - "_3"
+    - "_4"
+    - "idp"
+    - "refugee"
+    - "returnee"
+    - " cluster "
+    - " commune "
 
-food_security_default:
-  scrapers_with_defaults:
-    - "food_security_ch"
-  input:
-    - "chtype"
-    - "reference_label"
-    - "reference_year"
-    - "population"
-    - "phase1"
-    - "phase2"
-    - "phase3"
-    - "phase4"
-    - "phase5"
-    - "phase35"
-  list:
-    - "chtype"
-    - "reference_label"
-    - "reference_year"
-    - "population"
-    - "phase1"
-    - "phase2"
-    - "phase3"
-    - "phase4"
-    - "phase5"
-    - "phase35"
-  output:
-    - "ipc_type"
-    - "reference_period_months"
-    - "reference_period_year"
-    - "population_total"
-    - "population_phase1"
-    - "population_phase2"
-    - "population_phase3"
-    - "population_phase4"
-    - "population_phase5"
-    - "population_phase3+"
-  output_hxl:
-    - "#ipc+type"
-    - "#date+months"
-    - "#date+years"
-    - "#affected+ch+food+analysed+num"
-    - "#affected+ch+food+p1+num"
-    - "#affected+ch+food+p2+num"
-    - "#affected+ch+food+p3+num"
-    - "#affected+ch+food+p4+num"
-    - "#affected+ch+food+p5+num"
-    - "#affected+ch+food+p3plus+num"
+  adm1_only:
```

Comment:
Nice - I'll include this same logic in WFP where I think I have some similar issues.

Reply:
Thanks! Yes we may have to make it more strict given the inconsistency in usage of admin levels